docs/project-model/overview.md

```diff
@@ -0,0 +1,8 @@
+# .Net core project system
+
+Since .net core can be used on multiple platform it cannot rely on a single editor or enviroment. Because of this, and to prevent every editor to make their own implementation, the project system for .net core is now part of the platform.
```

Comment:
This frames the project model as an evolution of something specific to an editor or environment.  While true (to some extent), I don't think that's what people will be coming here to read about.  I think it would be better to describe what it is and highlight some of the "need to know" highlights of the project model.

In fact, I think we should hold off on this overall conceptual description right now - there's some churn going on internally right now such that what gets written today could end up being patently false a month from now.  We'll need to get the overall story straight before we can have a definitive conceptual doc describing the project model as a whole.